ACPs/236-continuous-staking/README.md

```diff
@@ -0,0 +1,154 @@
+| ACP           | 236                                                         |
+|:--------------|:------------------------------------------------------------|
+| **Title**     | Continuous Staking                                          |
+| **Author(s)** | Razvan Angheluta ([@rrazvan1](https://github.com/rrazvan1)) |
+| **Status**    | Proposed                                                    |
+| **Track**     | Standards                                                   |
+
+## Abstract
+
+This proposal introduces continuous staking for validators on the Avalanche P-Chain. Validators can stake their tokens
+continuously, allowing their stake to compound over time, accruing rewards once per specified cycle.
+
+## Motivation
+
+The current staking system on the Avalanche P-Chain restricts flexibility for stakers, limiting their ability to respond
+to changing market conditions or liquidity needs. Managing a large number of nodes is also challenging, as re-staking at
+the end of each period is labor-intensive, time-consuming, and poses security risks due to
+the required transaction signing. Additionally, tokens can remain idle at the end of a staking period
+until stakers initiate the necessary transactions to stake them again.
+
+## Specification
+
+Continuous staking introduces a mechanism that allows validators to remain staked indefinitely, without having to
+manually submit new staking transactions at the end of each period.
+
+Instead of committing to a fixed end time upfront, validators specify a cycle duration (period) when they submit an
+AddContinuousValidatorTx. At the end of each cycle, the validator is automatically re-staked for a new cycle of the same
+duration, unless the validator has submitted a StopContinuousValidatorTx. If a validator submits a
+StopContinuousValidatorTx during a cycle, the validator will continue validating until the end of the current cycle, at
+which point the validator exits and funds unlock. The minimum and maximum cycle lengths follow the same protocol
+parameters as before (`MinStakeDuration` and `MaxStakeDuration`).
+
+Delegators interact with continuous validators in the same way as with fixed-period validators, and the same constraints
+apply: a delegation period must fit entirely within the validator’s cycle. Delegators cannot delegate across multiple
+cycles, since there is no guarantee that a validator will continue validating after the current cycle.
```

Comment:
```suggestion
Delegator interaction remains unchanged, and the same constraints apply: a delegation period must fit entirely within the validator’s cycle. Delegators cannot delegate across multiple
cycles, since there is no guarantee that a validator will continue validating after the current cycle.
Essentially, it is not possible to delegate continuously. 
```